stdlib/2/os/path.pyi

```diff
@@ -4,7 +4,8 @@
 # based on http://docs.python.org/3.2/library/os.path.html
 # adapted for 2.7 by Michal Pokorny
 
-from typing import overload, List, Any, Tuple, BinaryIO, TextIO, TypeVar, Callable, AnyStr
+from typing import overload, List, Any, Tuple, BinaryIO, TextIO, TypeVar, Callable, AnyStr, Union
+EitherStr = Union[bytes, unicode]
```

Comment:
Use `_` prefix here (e.g. `_EitherStr`) so that `from os.path import *` doesn't import this alias.